lua/lscs/autorun/cl_forcepowers.lua

```diff
@@ -252,6 +252,7 @@ local function PlayerButtonDown( ply, button )
 	for _, name in pairs( Input ) do
 		for ID, power in pairs( ForcePowers ) do
 			if power.item.id == name then
+				if hook.Run( "LSCS:PlayerCanForce", ply, power ) == false then continue end
```

Comment:
inconsistency, the hook in lscs_forcepowers.lua passes an id, in this hook you are passing the item itself

Reply:
i might be wrong tho, need to test this as soon i am on the pc

Reply:
`local AllowForce = hook.Run( "LSCS:PlayerCanForce", ply, selectedF )`

Hook in lscs_forcepowers passes the selected force item